### PR TITLE
Remove some substeps from istep 20 and 21 as per ipl flow document

### PR DESCRIPTION
**Pull request by lkammath**

### CODE REVIEW

**Comment by lkammath** [comment]
Tested these changes.

File: src/istep/edbgIstep.C [hunk 1]
```diff
@@ -232,14 +232,10 @@ const edbgIPLTable::edbgIStep_t edbgIPLTable::cv_edbgIStepTable[] =
   {18,  13,                            "attn_listenall",                   EDBG_ISTEP_NOOP},
   {19,   1,                                 "prep_host",                   EDBG_ISTEP_NOOP},
   {20,   1,                         "host_load_payload",                   EDBG_ISTEP_HOST},
-  {20,   2,                        "host_load_complete",                   EDBG_ISTEP_HOST},
-  {21,   1,                         "host_micro_update",                   EDBG_ISTEP_NOOP},
-  {21,   2,                        "host_runtime_setup",                   EDBG_ISTEP_HOST},
+  {21,   1,                        "host_runtime_setup",                   EDBG_ISTEP_HOST},
+  {21,   2,                         "host_micro_update",                   EDBG_ISTEP_NOOP},
```
Comment (ojayanth) [inline src/istep/edbgIstep.C:236]: use "dummy_istep_name" instead of host_micro_update

Comment (lkammath) [reply src/istep/edbgIstep.C]: Done

**Comment by ojayanth** [comment]
Need to hold this PR . as discussed , looks like ecmd-pdbg is good.

**Comment by lkammath** [comment]
Yes. A phal/ipl change will fix this issue. ecmd-pdbg istep is in sync

**Comment by lkammath** [comment]
Closing this PR . A phal/IPL chang fixed the problem. no change is needed in the ecmd-pdbg istep numbering

**Pull request closed without merging**